org.opent2t.sample.lamp.superpopular/com.philips.hue.bulb/js/package.json

```diff
@@ -1,21 +1,31 @@
 {
-  "name": "opent2t-translator-com-philips-hue-bulb",
-  "version": "0.1.0",
-  "description": "Philips Hue Light Bulb",
-  "homepage": "http://opentranslatorstothings.org",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/opent2t/translators.git"
-  },
-  "main": "thingTranslator.js",
-  "dependencies": {
-    "node-hue-api": "~2.2.0"
-  },
-  "devDependencies": {
-    "optimist": "~0.6.1",
-    "readline": "~1.3.0",
-    "opent2t-onboarding-huehub": "*"
-  },
-  "author": "",
-  "license": "MIT"
+    "name": "opent2t-translator-com-hue-bulb",
```

Comment:
the name of the package is correct, can you update the folder name to align with this?  instead of com.philips.hue.bulb it should be com.hue.bulb